src/main/java/org/opensrp/repository/postgres/mapper/custom/xml/CustomOrganizationMapper.xml

```diff
@@ -96,6 +96,46 @@
       </foreach>
     </where>
   </sql>
-
+	<resultMap id="resultMap" type="org.opensrp.domain.Organization">    
+	    <id column="id" jdbcType="BIGINT" property="id" />
+	    <result column="identifier" jdbcType="VARCHAR" property="identifier" />
+	    <result column="active" jdbcType="BIT" property="active" />
+	    <result column="name" jdbcType="VARCHAR" property="name" />    
+	    <result column="date_deleted" jdbcType="TIMESTAMP" property="dateDeleted" />
+	    <result column="parent_id" jdbcType="BIGINT" property="partOf" />
+	    <result column="member_count" jdbcType="BIGINT" property="memberCount" />
+	    
+  	</resultMap>
+  	<sql id="query">
+		select o.id,o.identifier,o.date_deleted,o."name",o."type",o.parent_id,count(distinct(pr.id))as member_count 
```

Comment:
You should filter out deleted organization instead of selecting the `date_deleted`